test/integration_test.js

```diff
@@ -1430,6 +1430,56 @@ describe("Integration tests", function() {
               });
             });
 
+            describe("'at' retrieves a snapshot at a given timestam", () => {
+              let rec1, rec2, rec3;
+
+              beforeEach(() => {
+                return coll.listRecords()
+                  .then(_ => coll.createRecord({n: 1}))
+                  .then(({data}) => {
+                    rec1 = data;
+                    return coll.createRecord({n: 2});
+                  })
+                  .then(({data}) => {
+                    rec2 = data;
+                    return coll.createRecord({n: 3});
+                  })
+                  .then(({data}) => rec3 = data);
+              });
+
+              it("should handle creations", () => {
+                return coll.listRecords({at: rec1.last_modified})
+                  .should.eventually.become([
```

Comment:
I think it should return the same structure as `_paginatedList` : 
https://github.com/Kinto/kinto-http.js/blob/master/src/base.js#L410-L415